app/assets/stylesheets/administrate/components/_sidebar.scss

```diff
@@ -3,6 +3,7 @@
   max-width: 12em;
   overflow-y: auto;
   padding: 0 $base-spacing;
+  padding-bottom: $base-spacing;
```

Comment:
This addition means we are defining `padding` twice. Line 5 applies `0` to the `padding-bottom`, and Line 6 we overwrite that.

Instead, we can just update Line 5 to; `padding: 0 $base-spacing $base-spacing;`